PWGCF/JCorran/Tasks/jflucAnalysisTask.cxx

```diff
@@ -91,65 +86,65 @@ struct jflucWeightsLoader {
   Produces<aod::JWeights> output;
   void init(InitContext const&)
   {
-    //
-    if (!doprocessLoadWeights)
+    if (!doprocessLoadWeights && !doprocessLoadWeightsCF)
       return;
+    if (doprocessLoadWeights && doprocessLoadWeightsCF)
+      LOGF(fatal, "Only one weights loader process switch can be enabled at a time.");
     if (pathPhiWeights.value.substr(0, 8) == "local://") {
       pf = new TFile(pathPhiWeights.value.substr(8).c_str(), "read");
```

Comment:
Have in mind that the final source for the weights will be the CCDB which does not allow individual files but objects.